test/environments/norails/Gemfile

```diff
@@ -12,3 +12,4 @@ gem 'newrelic_rpm', :path => '../../..'
 
 gem 'pry', '~> 0.14.1'
 gem 'pry-nav'
+gem 'simplecov'
```

Comment:
Do we still need these gem declarations in the `test/environments/*` Gemfiles?

Reply:
Also, do they break anything for the tests that run on Ruby versions below 2.7?

Reply:
Testing this :)

Reply:
This has been addressed! Removed the SimpleCov gem from Rails versions that don't support Ruby versions above 2.7.0